shared/js/background/pixel.es6.js

```diff
@@ -0,0 +1,102 @@
+/**
+ *
+ * This is part of our tool for anonymous engagement metrics
+ * Learn more at https://duck.co/help/privacy/atb
+ *
+ */
+
+const load = require('./load.es6')
+const browserWrapper = require('./$BROWSER-wrapper.es6')
+const settings = require('./settings.es6')
+const parseUserAgentString = require('../shared-utils/parse-user-agent-string.es6')
+
+/**
+ *
+ * Fire a pixel
+ *
+ * @param {string} pixelName
+ * @param {...*} args - any number of extra data
+ *
+ */
+function fire () {
+    if (!arguments.length) return
+
+    let args = Array.prototype.slice.call(arguments)
+    const pixelName = args[0]
+
+    if (typeof pixelName !== 'string') return
+
+    const url = getURL(pixelName)
+
+    if (!url) return
+
+    args = args.slice(1)
+    args.push(getAdditionalParams())
+    const paramString = concatParams(args)
+
+    // Send the request
+    load.url(url + paramString)
+}
+
+/**
+ *
+ * Return URL for the pixel request
+ *
+ */
+function getURL (pixelName) {
+    if (!pixelName) return
+
+    const url = 'https://improving.duckduckgo.com/t/'
+    return url + pixelName
+}
+
+/**
+ *
+ * Return additional params for the pixel request
+ *
+ */
+function getAdditionalParams () {
+    const browserInfo = parseUserAgentString()
+    const browser = browserInfo.browser
+    const extensionVersion = browserWrapper.getExtensionVersion()
+    const atb = settings.getSetting('atb')
+    const result = {}
+
+    if (browser) result.browser = browser
```

Comment:
Can we add browser as a positional argument rather than a query string one? So something like:

```
/t/ep_chrome?foo=bar
/t/ept_off_firefox?foo=bar
```